pcsx2/CDVD/CDVDisoReader.cpp

```diff
@@ -16,7 +16,7 @@
 
 /*
  *  Original code from libcdvd by Hiryu & Sjeep (C) 2002
- *  Modified by Florin for PCSX2 emu
+ *  Modified by Florin, Weirdbeardgame for PCSX2 emu
```

Comment:
Adding a reset call and renaming a function hardly warrants adding your name to the list of people who translated libcdvd to pcsx2.